scripts/download-prebuilt.sh

```diff
@@ -41,7 +41,7 @@ set -e
 GITHUB_URL=https://github.com/timvideos/HDMI2USB-firmware-prebuilt/
 
 UPSTREAM_BRANCH=master
-UPSTREAM_REMOTE=$(git remote -v | grep fetch | grep 'timvideos/HDMI2USB-litex-firmware' | sed -e's/\t.*$//')
+UPSTREAM_REMOTE=$(git remote -v | grep fetch | grep 'timvideos/litex-buildenv' | sed -e's/\t.*$//')
```

Comment:
Could you change this to an or? I think it would be;
`grep 'timvideos/litex-buildenv|timvideos/HDMI2USB-litex-firmware'`

Reply:
Is the name of the repo important vs "find the timvideos" one?

does putting "fetch" in its own grep make it easier to read?  makes me edgy :p

have some options.  

$ git remote -v | 
grep -E 'timvideos/(litex-buildenv|HDMI2USB-litex-firmware) \(fetch\)' 
origin	https://github.com/timvideos/HDMI2USB-litex-firmware (fetch)

grep '\(fetch\)' |  grep -E 'timvideos/(litex-buildenv|HDMI2USB-litex-firmware)' 
origin	https://github.com/timvideos/HDMI2USB-litex-firmware (fetch)

grep 'timvideos/.*\(fetch\)' 
origin	https://github.com/timvideos/HDMI2USB-litex-firmware (fetch)

Reply:
I went with `grep '(fetch)' | grep -E 'timvideos/(litex-buildenv|HDMI2USB-litex-firmware)'`